cuesubmit/plugins/blender/README.md

```diff
@@ -0,0 +1,10 @@
+# OpenCue Blender addon
+
+The OpenCue Blender addon provides a standalone job submission interface 
+from within Blender. This directory consists of the following:
+
+- addon source code under the `OpenCue`
```

Comment:
Don't you mean `OpenCue-blender`?

Reply:
Yes, my bad. Will fix naming references throughout.